quadra/datasets/anomaly.py

```diff
@@ -213,19 +214,19 @@ def __init__(
         self.split = self.samples.split.unique()[0]
 
         self.crop_area = crop_area
-        self.valid_area_mask: Optional[np.ndarray] = None
+        self.valid_area_mask: np.ndarray | None = None
 
         if valid_area_mask is not None:
             if not os.path.exists(valid_area_mask):
                 raise RuntimeError(f"Valid area mask {valid_area_mask} does not exist.")
 
-            self.valid_area_mask = cv2.imread(valid_area_mask, 0) > 0
+            self.valid_area_mask = cv2.imread(valid_area_mask, 0) > 0  # type: ignore[operator]
```

Comment:
I think the problem here is that 'cv2.imread()' can still return None if the path is not available. I have seen this multiple times. We can make sure that we found the image path correctly.

Reply:
Makes sense